src/api/app/models/notification_bs_request.rb

```diff
@@ -0,0 +1,30 @@
+class NotificationBsRequest < Notification
```

Comment:
Is there a reason why all the new models are prefixed with `notification_` instead of being created in a subfolder (`.../models/notification/bs_request.rb` for instance)?

Reply:
No...